creduce/cvise_reduce.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
`#! /bin/sh` unless we *have* to use bash (which I don't think we do in this file).

I also wondered if this should be named `cvis.example.sh" or simukar?

Reply:
updated 👉  0940e4b7173b60a87a0cf5583add4faedba94afc